src/python/grpcio/grpc/__init__.py

```diff
@@ -905,6 +905,21 @@ def add_secure_port(self, address, server_credentials):
     raise NotImplementedError()
 
   @abc.abstractmethod
+  def add_shutdown_handler(self, shutdown_handler):
+    """Adds a shutdown handler to be called on server shutdown.
```

Comment:
Drop the first "handler" in this sentence - the term "shutdown handler" isn't defined until the reader reads the rest of the sentence and infers that being called on server shutdown is what makes a handler a "shutdown handler".